website/docs/Use-Cases/enhanced_inference.md

```diff
@@ -168,20 +168,44 @@ Total cost: 0.00027
 
 ## Caching
 
-API call results are cached locally and reused when the same request is issued. This is useful when repeating or continuing experiments for reproducibility and cost saving. It still allows controlled randomness by setting the "cache_seed" specified in `OpenAIWrapper.create()` or the constructor of `OpenAIWrapper`.
+API call results are cached locally and reused when the same request is issued. This is useful when repeating or continuing experiments for reproducibility and cost saving.
+
+Starting version 0.2.8, a configurable context manager allows you to easily configure
+the cache, using either DiskCache or Redis.
+All LLM agents inside the context manager will use the same cache.
 
 ```python
-client = OpenAIWrapper(cache_seed=...)
-client.create(...)
+from autogen.cache.cache import Cache
+
+with Cache.redis(cache_seed=42, redis_url="redis://localhost:6379/0") as cache_client:
+    user.initiate_chat(assistant, message=coding_task, cache_client=cache_client)
+
+with Cache.disk(cache_seed=42, cache_dir=".cache") as cache_client:
+    user.initiate_chat(assistant, message=coding_task, cache_client=cache_client)
 ```
 
+You can control the randomness by setting the `cache_seed` parameter.
+
+### Turnning off cache
+For backward compatibility, DiskCache is always enabled by default
+with `cache_seed` set to 41. To fully disable it, set `cache_seed` to None.
+
 ```python
+# You turn off cache directly through the agent,
+assistant = AssistantAgent(
+    "coding_agent",
+    llm_config={
+        "cache_seed": None,
+        "config_list": OAI_CONFIG_LIST,
+        "max_tokens": 1024,
+    },
+)
+
```

Comment:
There is a problem with these. The context for this page is about the enhanced inference (`oai`) and not about the agent chat interface (`agentchat`).
I suggest talking about the change in the `oai` part here, and provide a link to the agent chat level caching.

Reply:
Updated the docs. Now agentchat doc only talks about agent chat interface, and enhanced inference doc only talks about OpenAIWrapper interface.